pkg/microservice/aslan/core/delivery/service/docker.go

```diff
@@ -0,0 +1,57 @@
+/*
```

Comment:
is it required?

Reply:
merge issue, this file is deleted is pr 698